### PR TITLE
Fix example for messages_path method [SKIP CI]

### PR DESCRIPTION
**Pull request by artofhuman**

### CODE REVIEW

File: lib/hanami/validations.rb [hunk 1]
```diff
@@ -204,7 +204,7 @@ def messages(type)
       #   class Signup
       #     include Hanami::Validations
       #
-      #     messages :i18n
+      #     messages_path 'config/messages.yml'
```
Comment (cllns) [inline lib/hanami/validations.rb]: Do we need to specify `    messages :yaml`  if we do `messages_path`?

Comment (artofhuman) [inline lib/hanami/validations.rb]: I think no
https://github.com/hanami/validations/blob/master/test/messages_test.rb#L6-L9

Comment (solnic) [inline lib/hanami/validations.rb]: If you provide a path then it'll be used to load custom messages that will be merged with the builtin ones thus it shouldn't be required. If it is, I'd say it's probably a bug.

Comment (jodosha) [inline lib/hanami/validations.rb]: @cllns @artofhuman @solnic It isn't required. `:yaml` is the default (as in `dry-validation`).

**Pull request merged**